evm/vm_state.py

```diff
@@ -236,6 +245,34 @@ def get_transaction_context_class(cls):
     #
     # Execution
     #
+
+    def do_call(self, transaction):
+
+        _transaction = transaction
+
+        _transaction.v = DEFAULT_DO_CALL_V
+        _transaction.s = DEFAULT_DO_CALL_S
+        _transaction.r = DEFAULT_DO_CALL_R
+
+        snapshot = self.snapshot()
+        try:
+            with self.mutable_state_db() as state_db:
+
+                if not hasattr(_transaction, "get_sender"):
+                    # max out eth in random hardcoded address. what could go wrong?
+                    _transaction.get_sender = \
+                        lambda: to_bytes(hexstr=DEFAULT_DO_CALL_SENDER)
+                    _transaction.sender = to_bytes(hexstr=DEFAULT_DO_CALL_SENDER)
+
+                state_db.set_balance(transaction.sender, UINT_256_MAX, validate=False)
```

Comment:
This appears to be the only call site which uses the `validate=...` API.  It's not clear why it would be needed here.  Both of these values appear to be valid.